### PR TITLE
Added rotate90 icon

### PR DESCRIPTION
**Pull request by mateuszs**
image

### CODE REVIEW

File: src/images/icons/rotate_90.svg [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="512px" height="512px" viewBox="0 0 512 512" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (Malinicz) [inline src/images/icons/rotate_90.svg:2]: we need to format that here, take a look at other icons - it should have the same format

File: src/images/icons/rotate_90.svg [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                        <g transform="translate(-64.000000, -64.000000)" id="Rectangle">
+                            <rect x="0" y="0" width="512" height="512"></rect>
+                        </g>
+                    </g>
+                </g>
+            </g>
+            <rect id="Rectangle" x="0" y="0" width="512" height="512"></rect>
+        </g>
+    </g>
+</svg>
```
Comment (Malinicz) [inline src/images/icons/rotate_90.svg]: add line break

File: src/images/icons/rotate_90.svg [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (matzimowski) [inline src/images/icons/rotate_90.svg:1]: i see that it's not cleared at all from unnecessary markup, maybe u could pass it through https://jakearchibald.github.io/svgomg/
that's the good practise we try to keep.
u can take a look at other icons how "clean" they are

File: src/components/icons/Icon.jsx [hunk 4]
```diff
@@ -106,7 +106,8 @@ export type IconTypeType =
   | 'triangle'
   | 'pi'
   | 'quote'
-  | 'spark';
+  | 'spark'
+  | 'rotate_90';
 
 export type IconColorType =
```
Comment (matzimowski) [inline src/components/icons/Icon.jsx:112]: u have to define it in `const TYPE` too.

then I suggest to run `yarn watch` and try to actually find the icon in the style-guide website 😉

File: src/images/icons/rotate_90.svg (hunk 1 above)
Comment (Malinicz) [reply src/images/icons/rotate_90.svg]: you can take a look at `CONTRIBUTING.md` => `### Adding icons process`

File: src/components/icons/Icon.jsx [hunk 5]
```diff
@@ -106,7 +106,8 @@ export type IconTypeType =
   | 'triangle'
   | 'pi'
   | 'quote'
-  | 'spark';
+  | 'spark'
```
Comment (Malinicz) [inline src/components/icons/Icon.jsx:109]: add it also to `TYPE` constant

**Comment by matzimowski** [comment]
@mateuszs could u provide a screenshot of how this icon looks in a description, please?

File: src/images/icons/rotate_90.svg [hunk 6]
```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="512" height="512" viewBox="0 0 512 512">
+  <g fill="none" fill-rule="evenodd">
```
Comment (Malinicz) [inline src/images/icons/rotate_90.svg:2]: can you check if it works like that?

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
   <title>rotate 90</title>
    <path fill="#000" d="M404.877447,134.763523 L404.877447,106.764862 C404.877447,94.8913075 414.50274,85.2658897 426.376142,85.2658897 C438.249543,85.2658897 447.874837,94.8913075 447.874837,106.764862 L447.874837,191.760799 C447.874837,203.760225 437.875444,212.759795 426.876112,212.759795 L340.881331,212.759795 C333.379209,212.759795 326.446962,208.757409 322.6959,202.260297 C318.944839,195.763185 318.944839,187.758413 322.6959,181.261301 C326.446962,174.764188 333.379209,170.761802 340.881331,170.761803 L378.879025,170.761803 C338.198617,111.363412 260.865745,89.5498396 195.142592,118.93458 C129.41944,148.319321 94.1083529,220.49587 111.246404,290.42002 C128.384455,360.34417 193.068722,408.011737 264.930808,403.674171 C336.792895,399.336605 395.274205,344.234814 403.877507,272.756927 C404.98201,260.883372 415.50268,252.153342 427.376081,253.257859 C439.249483,254.362376 447.9794,264.883181 446.874898,276.756736 C436.986111,367.699461 364.307237,439.034101 273.196835,447.222533 C182.086432,455.410966 97.8530607,398.178616 71.9074073,310.456466 C45.9617538,222.734316 85.5109771,128.889741 166.416749,86.1994942 C247.322521,43.5092475 347.110914,63.831785 404.877447,134.763523 Z M201.430939,211 C234.911601,211 249,236.811321 249,268.082728 C249,301.339622 230.270718,325 197.618784,325 C184.193369,325 172.922652,320.532656 165.132596,313.583454 L165.132596,313.583454 L175.574585,295.548622 C181.541436,301.174166 187.839779,304.152395 197.618784,304.152395 C215.850828,304.152395 225.298342,288.930334 225.298342,273.046444 L225.298342,273.046444 L225.298342,270.564586 C219.994474,278.010159 208.889502,284.793904 196.790055,284.793904 C176.900552,284.793904 159,272.384616 159,248.889696 C159,227.380261 176.237568,211 201.430939,211 Z M306.5,211 C337.822463,211 352,239.458636 352,267.917271 C352,296.375908 337.822463,325 306.5,325 C275.012681,325 261,296.375908 261,267.917271 C261,239.458636 275.012681,211 306.5,211 Z M305.581749,232 C289.885932,232 284,247.652173 284,267.91762 C284,288.183066 289.885932,304 305.581749,304 C321.114068,304 327,288.183066 327,267.91762 C327,247.652173 321.114068,232 305.581749,232 Z M202.51181,232 C189.944882,232 183,240.666667 183,248.333334 C183,260 193.086614,265 203.338582,265 C211.606299,265 219.708661,261.166667 225,254.666667 C224.007874,244 218.055118,232 202.51181,232 Z"  fill-rule="evenodd" />
</svg>
```
changes:
width/height removed
<g> tag removed
fill-rule added to <path>

Comment (mateuszs) [reply src/images/icons/rotate_90.svg:2]: Yes, it works. I checked it on dev mode and also in built version. Screen I provided is from style-guide in built version

File: src/images/icons/rotate_90.svg [hunk 7]
```diff
@@ -0,0 +1,4 @@
...
+</svg>
```
Comment (Malinicz) [inline src/images/icons/rotate_90.svg:4]: add new line :)

**Pull request merged**